MSSQL/OMOPLoader.sql

```diff
@@ -171,7 +171,7 @@ DECLARE @SQL as varchar(4000)
 IF  OBJECT_ID(N'.[dbo].[loyalty_cohort_patient_summary]','U') IS NOT NULL ---Need to put in a DB name before .[dbo] for your datamart.
 SET @SQL='
 create view i2b2loyalty_patients as
-(select patient_num,cast(''2010/7/1'' as datetime) period_start,cast(''2014/7/1'' as datetime) period_end from PCORI_Mart..loyalty_cohort_patient_summary where filter_set & 61511 = 61511 and patient_num in (select patient_num from i2b2patient))'

```

Comment:
Again I didn't follow my own rule. Should probably just rip out i2b2loyalty_patients per my email.

Reply:
For the moment I reverted to where it was the latest commit. I'll submit another pull request that removes it as discussed in the email.